Mathlib/Topology/PartialHomeomorph.lean

```diff
@@ -673,14 +686,12 @@ theorem isOpen_inter_preimage_symm {s : Set α} (hs : IsOpen s) : IsOpen (e.targ
 /-- The image of an open set in the source is open. -/
 theorem image_isOpen_of_isOpen {s : Set α} (hs : IsOpen s) (h : s ⊆ e.source) :
     IsOpen (e '' s) := by
-  have : e '' s = e.target ∩ e.symm ⁻¹' s := e.toPartialEquiv.image_eq_target_inter_inv_preimage h
-  rw [this]
-  exact e.continuousOn_symm.isOpen_inter_preimage e.open_target hs
+  apply e.isOpen_image_of_subset_source hs h
 #align local_homeomorph.image_open_of_open PartialHomeomorph.image_isOpen_of_isOpen
 
 /-- The image of the restriction of an open set to the source is open. -/
 theorem image_isOpen_of_isOpen' {s : Set α} (hs : IsOpen s) : IsOpen (e '' (e.source ∩ s)) :=
```

Comment:
Are you saying this duplicates a lemma?

If we're keeping it it should be renamed to `isOpen_image_source_inter_of_isOpen`.

Reply:
Renamed. (I meant a different lemma, clarified now.)